core/src/main/java/greencity/config/ScheduleConfig.java

```diff
@@ -134,4 +134,16 @@ public void chooseNewFactOfTheDay() {
     public void scheduleDeleteDeactivatedUsers() {
         userRepo.scheduleDeleteDeactivatedUsers();
     }
+
+    /**
+     * Every day at 00:00 deletes from the database users
+     * that have status 'CREATED' and have not activated the account within 24 hours.
```

Comment:
So If I start my registration at 23:55, I won't be able to finish it at 00:00.

I believe you should delete those, whose creation date is before yesterday.

Reply:
The idea was the following:
If we create the User (gets `CREATED` status) then the appropriate record will be created in `restore_password_email`  (records in this table are automatically deleted after 86400000 seconds = 24 hours after they were created). So I was checking to see if we have a record in this table for our user. If there were no records - mean 24 hours have passed and we can delete this user.
***
But for now, I redid it to check `date_of_registration` as you suggested.

Reply:
When does the `restore_password_email` table is cleaned?

Reply:
![image](https://user-images.githubusercontent.com/45639641/93747331-399c5380-fbff-11ea-81ba-270df853bceb.png)

Reply:
Shouldn't this be triggered together with `userRepo.scheduleDeleteCreatedUsers()` ?

Reply:
Ah, that is different. Never mind.